repose-aggregator/components/filters/scripting-filter/build.gradle

```diff
@@ -0,0 +1,20 @@
+dependencies {
+    compile project(":repose-aggregator/core/core-lib")
```

Comment:
It's not. It was in the branch I started on, and I just didn't remove it.

Although, it was apparently providing jcl-over-slf4j transitively.